### ISSUE TITLE
Revisiting the title of the paper

### ISSUE THREAD

**Issue by bsilverajan** [TakeshiTakahashi/smartVesselPaper#1]
The current title of the paper is "Toward fully automated smart vessels: designing network and its data-oriented security".

I think it would be a good idea not to think only about fully automated smart vessels, but also smart vessels with a reduced crew or remote controlled vessels. This gives us a wider range of possible security threats to consider (for example, tampering or infected devices brought onboard by the crew, and so on). However, we should not need to consider many kinds of security threats. 

How would you feel if we have, for example, a broader title and then give the scope in the paper itself?

**Comment by TakeshiTakahashi** [comment]
Yes, I agree to work on semi-automated vehicles. The sentences put in the initial version are there only for the purpose of expediting discussion, so please by all means, make any necessary changes on the paper.
